Usuarios listos y gestion comienzo de partida

jmarindiez opened the pull request:
Closes #115

jmarindiez committed:
Start game

jmarindiez committed:
cerrar popup

jmarindiez committed:
preparing game y start

jmarindiez committed:
no usado

jmarindiez committed:
Format

xzebra committed:
fix socket undefined

xzebra committed:
reorder and group

xzebra committed:
initialUsers for createGame

xzebra committed:
fix tests

xzebra committed:
get rid of string numbers and format

xzebra commented:
Faltaria gestionar cuando el lider abandona la partida y le toca al cliente ser el lider. Deberia cambiar al popup del boton de empezar.

jmarindiez committed:
Popup generico permite distintas acciones en el cierre

jmarindiez committed:
Gestion cambio de propietario

jmarindiez commented:
* No actualiza bien el numero de usuarios en la partida cuando se va el propietario.

    1. Hay 2 usuarios en partida (propietario y otro). 
    2. El propietario abandona.
    3.  Se traspasa el "poder" al otro usuario. (Queda solo un usuario, pero... en el popup de empezar aparecen 2!!)

*  Cuando el propietario abandona la partida desde `/match` se captura el game_owner por lo que vuelve a aparecer el popup de empezar en medio de la partida.

jmarindiez committed:
salida de partida

jmarindiez committed:
Merge branch 'master' of github.com:UNIZAR-30226-2021-07/web into usuarios-listos

jmarindiez committed:
spinners

jmarindiez committed:
Format

jmarindiez committed:
Mensajes que se adaptan al contenido

FCanfran committed:
fuera los socket on de app

FCanfran committed:
fixing socket bug on match

jmarindiez committed:
Merge branch 'master' of github.com:UNIZAR-30226-2021-07/web into estilo-mensajes

FCanfran committed:
obtencion de neuvo socket al volver a menu
desde partida.
Al refrescar se sale de partida, volviendo a menu, socket se habrá desconectado por lo que se pide uno nuevo. Por simplicidad al salir de partida también se pide nuevo socket

jmarindiez committed:
Mensajes del propio usuario a la derecha y los del servidor centrados

jmarindiez committed:
Merge branch 'master' of github.com:UNIZAR-30226-2021-07/web into usuarios-listos

jmarindiez committed:
Format

FCanfran committed:
arreglado el useEffect de joinPublicGamePopup

FCanfran committed:
format

jmarindiez committed:
Merge branch 'arreglos' of github.com:UNIZAR-30226-2021-07/web into arreglos

jmarindiez committed:
Eliminar logs

jmarindiez committed:
Fix lint

FCanfran committed:
intentando sacar la escucha de num_players waiting de los popups, para que sea un estado comun compartido por todos ellos

xzebra committed:
Merge branch 'usuarios-listos' into arreglos

xzebra committed:
Merge pull request #119 from UNIZAR-30226-2021-07/arreglos

fuera los socket on de app

FCanfran committed:
pre-merge

FCanfran committed:
Merge branch 'usuarios-listos' of github.com:UNIZAR-30226-2021-07/web into usuarios-listos

FCanfran committed:
no se hacerlo con contextos

FCanfran committed:
solucionado con contexto (?)

FCanfran committed:
format

FCanfran commented:
>     * No actualiza bien el numero de usuarios en la partida cuando se va el propietario.
>       
>       1. Hay 2 usuarios en partida (propietario y otro).
>       2. El propietario abandona.
>       3. Se traspasa el "poder" al otro usuario. (Queda solo un usuario, pero... en el popup de empezar aparecen 2!!)
> 
>     * Cuando el propietario abandona la partida desde `/match` se captura el game_owner por lo que vuelve a aparecer el popup de empezar en medio de la partida.

Falta de arreglar el segundo bug, el primero está provisionalmente arreglado.

xzebra committed:
Merge branch 'usuarios-listos' into estilo-mensajes

xzebra committed:
remove unused

xzebra committed:
update tests to add context

xzebra committed:
format tests

xzebra committed:
Merge pull request #118 from UNIZAR-30226-2021-07/estilo-mensajes

Estilo mensajes

xzebra commented:
En la branch `estilo-mensajes` que acabo de mergear he aprovechado para actualizar los tests que dependían de contexto, no se si estarán todos cubiertos.

FCanfran commented:
Closes #24

FCanfran committed:
sigo igual, propuesta de alternativas

FCanfran committed:
arreglado bug 2 de forma provisional pero muy decente

FCanfran committed:
formateo

FCanfran committed:
comentados los tests provisionalmente

FCanfran committed:
format

FCanfran committed:
linter

xzebra reviewed (approved):
Vale, los bugs mencionados están solucionados, aunque he encontrado otro. Creo un issue con la explicación.
